actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -88,13 +92,15 @@
 
     private CompletableFuture<Void> hostingActive = new Task<>();
 
-    private int maxLocalAddressCacheCount = 10_000;
-
     private NodeSelectorExtension nodeSelector;
 
-    public Hosting()
+    public Hosting(final int maxLocalAddressCacheCount, final long defaultActorTTL)
```

Comment:
An issue that has come up in a use case is that the local cache is keeping reference to an actor after it has been deactivated. In these cases when a node (A) has a cached reference of the actor that points to a node (B) and sends the message that way. The node (B) is no longer hosting the actor so has to check the dist cache. It finds the actor is now hosted on node (C) and has to forward the message to that node. It then sends back a moved message to node (A) to invalidate its local cache.

In these cases it is better to invalidate the entry earlier and go to the dist cache to find the actor instead of using the local cache.